docs/authentication-backends.md

```diff
@@ -50,23 +46,23 @@ To enable it:
    docs](https://www.netlify.com/docs/authentication-providers/#using-an-authentication-provider).
 2. Add the following lines to your `config.yml` file:
 
-   ``` yaml
-   backend:
-     name: github
-     repo: owner-name/repo-name # Path to your Github repository
-   ```
+    ``` yaml
+    backend:
+      name: github
+      repo: owner-name/repo-name # Path to your Github repository
+    ```
 
+
+### External OAuth Clients:
 If you would like to facilitate your own OAuth authentication rather than use Netlify's service, you
-can use one of the community maintained providers below, and feel free to [submit a pull
-request](https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md) if you'd like to add
-yours!
+can use one of the community-maintained projects below. Feel free to [submit a pull request](https://github.com/netlify/netlify-cms/blob/master/CONTRIBUTING.md) if you'd like to add yours!
+
```

Comment:
On re-reading, this paragraph made more sense under the External OAuth Clients heading than before it.